packages/office-ui-fabric-react/src/components/pickers/AutoComplete/AutoComplete.tsx

```diff
@@ -0,0 +1,23 @@
+/* tslint:disable */
+import * as React from 'react';
+import { css } from '../../../Utilities';
+/* tslint:enable */
+import { BasePicker } from '../BasePicker';
+import { IBasePickerProps } from '../BasePicker.types';
+import { SearchBox } from '../../../SearchBox';
+
+export interface ISuggestedItem {
```

Comment:
Is this suppose to be defined twice?  In the api.ts and in this file?

Reply:
@darewreck54 , I was trying put implement a concept on the existing component to get it working for a very basic case and then enhance the same from there based on feedback/comments. 

The problem was that the main build is failing for me so unable to produce a working version. It will take some time for me to actually, fix it in my local.

Reply:
@bishwenduk029 Is there a way to bring in the changes for me locally to test it?  Or do need to wait for you to address the fixes before i can see it off your branch